test/integration/contract-aci.js

```diff
@@ -304,6 +304,45 @@ describe('Contract instance', function () {
     result.callerId.should.be.equal(onAccount)
   })
 
+  describe('Transaction options', async () => {
+    it('deploys with no options', async () => {
+      const testContract = await aeSdk.getContractInstance({
+        source: testContractSource, filesystem
+      })
+      const deployInfo = await testContract.deploy(['test', 1, 'hahahaha'])
```

Comment:
Deployment with no options used multiple times in SDK tests, I don't think that it is needed to be tested explicitly.

Reply:
Sure